Reference/V9-Config/RuntimeMinificationSettings/index.md

```diff
@@ -15,45 +15,49 @@ This section allows you to configure the runtime minifications (defaults shown),
 "Umbraco": {
   "CMS": {
     "RuntimeMinification": {
-      "UseInMemoryCache": false,
-      "CacheBuster": "Version"
+      "UseInMemoryCache": true,
+      "CacheBuster": "Timestamp"
     }
   }
 }
 ```
+## Use 'in memory' cache
 
-## Use in memory cache
-
-Specifies if Smidge should use an in memory cache or not.
+This setting determines whether Smidge should save it's cached output in memory, or in a file on disk. If set to false, then the folder will be created at the wwwroot of your Umbraco site in a folder called 'Smidge'/
 
 ## Cache buster
 
-Specifies what type of cache buster to use, the options are:
+This is the setting that determines how Smidge should break it's cache.
+
+The options are:
 
-* Version - Caches will be busted every time the version string changes.
+* Version - Caches will be busted every time the version string changes in your appsettings.json file
```

Comment:
appsettings is only one way to manage configuration, possibly beter to mention "configuration" with a link to [Managing configuration](https://our.umbraco.com/documentation/Reference/V9-Config/#managing-configuration)